api/ContentIndex.json

```diff
@@ -0,0 +1,192 @@
+{
+  "api": {
+    "ContentIndex": {
+      "__compat": {
+        "support": {
+          "chrome": {
+            "version_added": false
+          },
+          "chrome_android": {
+            "version_added": "84"
+          },
+          "edge": {
+            "version_added": "84"
+          },
+          "firefox": {
+            "version_added": false
+          },
+          "firefox_android": {
+            "version_added": false
+          },
+          "ie": {
+            "version_added": false
+          },
+          "opera": {
+            "version_added": false
+          },
+          "opera_android": {
+            "version_added": false
```

Comment:
It's likely it will be in Opera for Android 60. How can we remember to update this later?

Reply:
I've filed https://github.com/mdn/browser-compat-data/issues/6601